laser/laser_class.py

```diff
@@ -0,0 +1,14 @@
+import RPi.GPIO as GPIO
```

Comment:
Couldn't this just become
```suggestion
from RPi import GPIO
```